graph/src/components/store.rs

```diff
@@ -135,16 +135,14 @@ impl EntityQuery {
 #[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
 #[serde(rename_all = "lowercase")]
 pub enum EntityChangeOperation {
-    /// A new entity was added.
-    Added,
-    /// An existing entity was updated.
-    Updated,
+    /// An entity was added or updated
```

Comment:
The commit message states `The distinction between [Added and Updated] was meaningless in the code, and with the way we now generate store events, we can no longer say whether a change adds or updates an entity`.

I have no objections to merging the two variants into one but I'd disagree with the second half of that statement: through the `op_id` in the history table, we _can_ tell whether a `EntityOperation::Set` was an addition or an update. In fact, we rely on this to generate entity changes of type `'removed'` when building store events for reverted blocks.

Reply:
I was referring to how we now generate EntityChanges in the app: at that point we do not know whether an operation will lead to an insert or update. You are of course right that we know from the history table. I'll clarify the commit message.